discord/command.go

```diff
@@ -33,9 +33,15 @@ type Command struct {
 	// GuildID is the guild id of the command, if not global.
 	GuildID GuildID `json:"guild_id,omitempty"`
 	// Name is the 1-32 lowercase character name matching ^[\w-]{1,32}$.
-	Name string `json:"name"`
+	Name              string         `json:"name"`
+	NameLocalizations CommandLocales `json:"name_localizations,omitempty"`
 	// Description is the 1-100 character description.
-	Description string `json:"description"`
+	Description              string         `json:"description"`
+	DescriptionLocalizations CommandLocales `json:"description_localizations,omitempty"`
+	// The fields NameLocalized, DescriptionLocalized are
+	// populated only when data is received.
+	NameLocalized        string `json:"name_localized,omitempty"`
```

Comment:
These are better named `LocalizedName` and `LocalizedDescription`, I think. The JSON tags stay the same.